content/guides/references/migration-guide.md

```diff
@@ -2,14 +2,275 @@
+#### After - Cypress 10 and `mount`
+
+```js
+import { useCounter } from ‘./useCounter’
+
+it('increments the count', () => {
+  function Counter () {
+    const { count, increment } = useCounter()
+    return (
+      <>
+        <h1 name="count">Count is {{ count }}</h1>
+        <button onClick={increment}>Increment</button>
+      </>
+    )
+  }
+
+  cy.mount(<Counter />).then(() => {
+    cy.get('[name="count"]').should('contain', 0)
+      .get('button').click()
+      .get('[name="count"]').should('contain', 1)
+  })
+})
+```
+
+### React - `unmount` Removed
+
+`unmount` from `cypress/react` has been removed. Read more about the rationale
+[here](https://www.cypress.io/blog/2022/11/04/upcoming-changes-to-component-testing/).
+We recommend using the API React provides for unmount components,
+[unmountComponentAtNode](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode).
+
+#### Before - Cypress 10 and `unmount`
+
+```js
+import { unmount } from ‘cypress/react’
+
+it('calls the prop', () => {
+  cy.mount(<Comp onUnmount={cy.stub().as('onUnmount')} />)
+  cy.contains('My component')
+
+  unmount()
+
+  // the component is gone from the DOM
+  cy.contains('My component').should('not.exist')
+  cy.get('@onUnmount').should('have.been.calledOnce')
+})
+```
+
+#### After - Cypress 11 and `unmountComponentAtNode`
+
+```js
+import { getContainerEl } from 'cypress/react'
+import ReactDom from 'react-dom'
+
+it('calls the prop', () => {
+  cy.mount(<Comp onUnmount={cy.stub().as('onUnmount')} />)
+  cy.contains('My component')
+
+  cy.then(() => ReactDom.unmountComponentAtNode(getContainerEl()))
+
+  // the component is gone from the DOM
+  cy.contains('My component').should('not.exist')
+  cy.get('@onUnmount').should('have.been.calledOnce')
+})
+```
+
+### Vue - `mountCallback` Removed
+
+`mountCallback` from `cypress/vue` has been removed. Read more about the
+rationale
+[here](https://www.cypress.io/blog/2022/11/04/upcoming-changes-to-component-testing/).
+We recommend using `mount`.
+
+#### Before - Cypress 10 and `mountCallback`
+
+```js
+import { mountCallback } from ‘cypress/vue’
+
+beforeEach(mountCallback(MessageList))
+
+it('shows no messages', () => {
+  getItems().should('not.exist')
+})
+```
+
+#### After - Cypress 11 and `mount`
+
+```js
+beforeEach(() => cy.mount(MessageList))
+
+it('shows no messages', () => {
+  getItems().should('not.exist')
+})
+```
+
+### Angular - Providers Mounting Options Change
+
+There is one breaking change for Angular users in regards to providers. In
+Cypress 10 we took any providers passed as part of the Mounting Options and
+overrode the component providers via the `TestBed.overrideComponent` API.
+
+In Cypress 11, providers passed as part of the Mounting Options will be assigned
+at the module level using the `TestBed.configureTestingModule` API.
+
+This means that module-level providers (resolved from imports or
+`@Injectable({providedIn: 'root' })` can be overridden but providers specified
```

Comment:
Should we add a space before "providedIn"?

Reply:
Yep... we should really set up linting for our docs.